helm/hpcc/templates/roxie.yaml

```diff
@@ -222,7 +222,7 @@ kind: ConfigMap
 
 {{- $_ := set $commonCtx "instanceNames" list -}}
 {{ if $roxie.serverReplicas -}}
-{{ $_ := set $commonCtx "instanceNames" (list $servername) -}}
+{{ $_ := set $commonCtx "instanceNames" (list $servername) }}
```

Comment:
This was causing the complaint about metadata.name
It didn't seem to change the output from helm template though, so I'm not quite sure what is going on